### PR TITLE
[client] Add my voted asks

### PR DESCRIPTION
**Pull request by dongyeon**
[client] add my voted asks

### CODE REVIEW

File: client/src/app/app-router.jsx [hunk 1]
```diff
@@ -8,18 +8,23 @@ var DefaultRoute = Router.DefaultRoute;
 
 var Master = require('./components/master.jsx');
 var Home = require('./components/home.jsx');
+var GetStarted = require('./components/get-started.jsx');
+var Feed = require('./components/feed.jsx');
+var Component = require('./components/component.jsx');
+var TextComponent = require('./components/text-component.jsx');
```
Comment (bongsok) [inline client/src/app/app-router.jsx:14]: Please remove all unused components.

File: client/src/app/components/voted-asks.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+var { Colors, Spacing, Typography } = mui.Styles;
+
+var MyVotedCardList = require('./voted-card-list.jsx');
+var WriteButton = require('./write-button.jsx');
+var Back = require('./svg-icons/back.jsx');
+
+var VotedAsks = React.createClass({
+
+  myVotedAsksItem : {
+  },
```
Comment (bongsok) [inline client/src/app/components/voted-asks.jsx:14]: Doesn't this object have to be made to global variant unlike others?

**Comment by Myoungsun-Lee** [comment]
+1

File: client/src/app/components/voted-asks.jsx (hunk 2 above)
Comment (dongyeon) [inline client/src/app/components/voted-asks.jsx:14]: I don't know what you mean exactly.
Could you explain it more detailedly??

**Comment by dongyeon** [comment]
Need to make another pull request.
So, I'll abandon this pull request.

**Pull request closed without merging**